### ISSUE TITLE
Run local files

### ISSUE THREAD

**Issue by arkany** [WestonThayer/Framed#3]
This may be outside of the scope of what you're looking to make, but the ability to run a framer file locally and have Framed spin up a local server would be the bees knees.

**Comment by WestonThayer** [comment]
Could you expand on the scenario? Is it running prototypes offline?

**Comment by arkany** [comment]
Yes, actually. I can't place any Framer file online for legal reasons, but I want to share my prototype with coworkers on Windows machines. I don't want to set up a local server for each of them. Framed looks to be a great way to share Framer prototypes with my fellow designers, research, etc, but I don't have the affordance of throwing it on the web.

**Comment by WestonThayer** [comment]
Cool, yeah that sounds like a great idea. As a workaround, if everybody is on the same company intranet, you can do this:
1. Start a server on your PC connected to the intranet. You can probably get to your prototype via `http://localhost:3000/` or something
2. Open a command prompt and run `ipconfig`
3. Grab your IPv4 address, something like `IPv4 Address. . . . . . . . . . . : 10.178.11.29`
4. Replace "localhost" with the IPv4, like `http://10.178.11.29:3000/` and send to your coworkers

As long as you keep the local server running on your machine (and they're on the same network as you), they should be able to access that link **only** internally, via the corporate intranet.

We do this all the time at Microsoft. I actually keep a web server under my desk and maintain a little home page as well.

**Comment by arkany** [comment]
You're absolutely right. I'd been doing something similar for running a camera module in my prototype. 

That being said, it would still be nice to have a better solution for remote workers, checking it out asynchronously, etc.

**Comment by joetheday** [comment]
I've tried two ways of using Framer on my PC, and neither load in Framed
Option 1: [atom-live-server](https://atom.io/packages/atom-live-server) - runs a server at `127.0.0.1:3000`
Option 2: [framer boilerplate](https://github.com/jchavarri/framer-boilerplate) - runs at `localhost:3000`

Using the IPv4 method does not work for either. Any ideas?

**Comment by WestonThayer** [comment]
@joetheday it's using a `WebView` under the hood. I would try making a little sample XAML app with a `<WebView>`, and loading up a `localhost` URL. That'll at least tell us if the issue is specific to Framed. Here's a sample to start from: https://github.com/Microsoft/Windows-universal-samples/tree/master/Samples/XamlWebView

**Comment by ajaxborges** [comment]
Was this ever figured out? I'd like to run a Framer project locally on my Android device. I also know how to create a simple Android app with a WebView.
